content/community/events.page.tsx

```diff
@@ -537,15 +635,15 @@ export default function Events() {
     const { name, checked } = event.target;
     setUpcomingFilters((prevFilters) => ({
       ...prevFilters,
-      [name.replace('-upcoming','')]: checked,
+      [name.replace("-upcoming", "")]: checked,
```

Comment:
There might be a problem with this code under Redocly. I'm seeing an error in the browser console, which might be related to this:

```
Warning: Prop `className` did not match. Server: "event-card conference-upcoming-upcoming" Client: "event-card conference-upcoming"
```